File: 0014-longest-common-prefix/0014-longest-common-prefix.cpp
class Solution {
public:
    string longestCommonPrefix(vector<string>& strs) {
        string res="";
        for(int j=0; j<strs[0].size(); j++){
            char c=strs[0][j];
            for(int i=1; i<strs.size(); i++){
                if(c!=strs[i][j]){
                    return res;
                }
            }
            res+=c;
        }
        return res;
    }
};

File: 0014-longest-common-prefix/0014-longest-common-prefix.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        res=""
        for j in range(0, len(strs[0])):
            c=strs[0][j]
            for i in range(1, len(strs)):
                if j>=len(strs[i]) or c!=strs[i][j]:
                    return res
            res+=c
        return res